### DISCUSSION TITLE
Typescript intellisense problem

### DISCUSSION THREAD

**Discussion by bbg** [chakra-ui/panda#1510] [Q&A] [answered]
Greetings, I set the outdir property to a node module and set the emitPackage property to true and everything works correctly, but typescript intellisense and auto import don't work. It behaves as if there is no package. How can I solve this problem.

```
import { defineConfig } from "@pandacss/dev";

export default defineConfig({
  preflight: true,
  include: ["./app/**/*.{js,jsx,ts,tsx}"],
  exclude: [],
  outExtension: "js",
  outdir: "@bionic-ui/styled-system",
  jsxFramework: "react",
  emitPackage: true,
  watch: true,
  forceConsistentTypeExtension: true,
});
```

![CleanShot 2023-10-10 at 16 15 38@2x](https://github.com/chakra-ui/panda/assets/13348876/bc96d893-5b0f-4f5e-9cf5-84f0db3a3493)

**Comment by astahmer** [comment, marked as answer]
https://panda-css.com/docs/overview/faq#why-is-my-ide-not-showing-styled-system-imports

> If you're not getting import autocomplete in your IDE, you may need to include the styled-system directory in your tsconfig.json file.
